Assets/_BForBoss/_Weapons/Scripts/Weapons/WeaponAnimationController.cs

```diff
@@ -82,10 +81,9 @@ public void WeaponFire(WeaponAnimationType type)
             }
         }
         
-        public void SwapWeapon(bool isUpwards)
+        public void SwapWeapon()
         {
-            var param = isUpwards ? SWAP_WEAPON_UP_PARAM : SWAP_WEAPON_DOWN_PARAM;
-            _weaponAnimator.SetTrigger(param);
+            _weaponAnimator.SetTrigger(SWAP_WEAPON_PARAM);
```

Comment:
Related to the refactor.
Previously, different animation clips needed for swapping weapons up/down, now only 1 is needed